plugins/vst_base/RemoteVstPlugin.cpp

```diff
@@ -1072,13 +1073,13 @@ void RemoteVstPlugin::saveChunkToFile( const std::string & _file )
 		const int len = pluginDispatch( 23, 0, 0, &chunk );
 		if( len > 0 )
 		{
-			int fd = open( _file.c_str(), O_WRONLY | O_BINARY );
-			if ( ::write( fd, chunk, len ) != len )
+			FILE* fp = F_OPEN_UTF8( _file, "wb" );
+			if ( fwrite( chunk, len, 1, fp ) != len )
```

Comment:
Only spotted this now: `fwrite`returns the `count` of elements successfully written, so in this case it will return `1` on success, not `len`. Therefore this condition is always leading to a false error message on success (except when coincidentally `len == 1`).

Reply:
Thank you! Changed to `fwrite( chunk, 1, len, fp )` in 0f3b41f590ed8d573ce3891f6502f211d1be176a.

Reply:
Also synced into `master` via 170a46e7a3b5bc3eb49999874344b82cda1cc53f.